<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ekle-Güncelle</string>
  </property>
  <widget class="QLabel" name="widgetResim">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>150</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>Placeholder Image Ekle</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>HES Kodu</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>TC Kimlik No</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="widgetTcKimlikNo"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="widgetDurum"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="widgetHesKodu"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Durum</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="widgetKaydet">
      <property name="text">
       <string>Kaydet</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="widgetIptal">
      <property name="text">
       <string>İptal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="widgetResimEkle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Resim Ekle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
